crowdin.yml

```diff
@@ -0,0 +1,9 @@
+files:
+  - source: /GUI/**/*.resx
+    ignore:
+      - Properties/
+      - Localization/
+      - HintTextBox.resx
```

Comment:
Is `HintTextBox.resx` excluded because it doesn't have any text resources? Or is there something else special about it?

Reply:
Sorry for the late answer, was a busy week(end).

I exluded it because Crowdin just copied the file for each translation because everything was supposedly translated when there is just nothing to translate.